### PR TITLE
[release-v1.0] Fix channel template spec type for sequence channel template field

### PR DESCRIPTION
**Pull request by matzew**
porting https://github.com/knative/eventing/commit/282bba706eaa13583788eca5f28288655ed945e0

and regenerating the manifest

### CODE REVIEW

File: openshift/release/knative-eventing-ci.yaml [hunk 1]
```diff
@@ -1510,6 +1510,9 @@ spec:
                   namespace:
                     description: 'Namespace of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/ This is optional field, it gets defaulted to the object holding it if left out.'
                     type: string
+              deadLetterSinkUri:
```
Comment (matzew) [inline openshift/release/knative-eventing-ci.yaml:1513]: @pierDipi the accidently merged PR also did _NOT_ have this, on our CI manifest

**Review by pierDipi — APPROVED** [review]
/lgtm

Feel free to unhold this as you like, I think it's a good bug fix that we get.

**Pull request merged**